WatchUI/Ibasic/IBasic.py

```diff
@@ -117,12 +124,13 @@ def set_log_message(
         if work_object == "Image":
             if type_of_messages == "Info":
                 logger.info(
-                    f"Image was saved to your output folder </p><img src={path_to_image}/>",
+                    f"<p>Image was saved to your output folder </p><img src={path_to_image}></img>",
```

Comment:
this bug caused that saved generated diff image was not loaded in the RF log file